packages/aws-cdk-lib/aws-ecs/lib/cluster.ts

```diff
@@ -263,6 +290,37 @@ export class Cluster extends Resource implements ICluster {
     Aspects.of(this).add(new MaybeCreateCapacityProviderAssociations(this, id));
   }
 
+  private updateKeyPolicyForEphemeralStorageConfiguration(key: IKey, clusterName?: string) {
```

Comment:
```suggestion
  \**
   * Applies policy to the target key for encryption.
   *
   * @see https://docs.aws.amazon.com/AmazonECS/latest/developerguide/fargate-create-storage-key.html
   *\
  private updateKeyPolicyForEphemeralStorageConfiguration(clusterName?: string) {
    const key = this._managedStorageConfiguration?.fargateEphemeralStorageKmsKey;
    if (!key) return;
```
We can use the global param to remove the `if` statement in the constructor.
Also, adds some documentation.